dali/test/python/test_operator_readers_webdataset_requirements.py

```diff
@@ -58,12 +60,10 @@ def test_skip_sample():
     extract_dir = generate_temp_extract(tar_file_path)
     equivalent_files = list(
         filter(
-            lambda s: int(s[s.rfind("/") + 1 : s.rfind(".")]) < 2500,
-            sorted(
-                glob(extract_dir.name + "/*"), key=lambda s: int(s[s.rfind("/") + 1 : s.rfind(".")])
-            ),
-        )
-    )
+            lambda s: int(s[s.rfind("/") + 1:s.rfind(".")]) < 2500,
```

Comment:
According to my flake8 it's not allowed, just tried it, reverting.

Reply:
It's a known bug in flake8.... known since 2015.... to make it even "funnier", black _adds_ those spaces and doesn't have an option not to :upside_down_face:

Reply:
Then:
```suggestion
            lambda s: int(s[s.rfind("/") + 1 : s.rfind(".")]) < 2500,  # noqa: 203
```
?

Reply:
I don't know if that's much better with the noqa, but done.